src/index.html

```diff
@@ -15,8 +15,97 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <div class="imgblock">
+        <img
+          class="imgblock"
+          src="https://s3-alpha-sig.figma.com/img/833f/9aab/47d660efd384318772b6b8040db831aa?Expires=1730073600&Key-Pair-Id=APKAQ4GOSFWCVNEHN3O4&Signature=GFjFD6-6KNqFBkswmu-BuTPJnotQ8-MD~XFWAHKRTFzBEQqKLe-4Cca9JX55r9scHqqYuwSS3TsIyJCh2iERYUgPUe5fszbMK7Tg0AgZZdstjYggVmO~lCJAm5RABG4y~Pd9KlhMIhkeNGbSjrNGNjL~pLc8xnAb1HFOb19z7kl6MngmQbQxT-1vuKf03PUIUzsEhIR8aicMa2KjYgomPp2TDMN9G7n8WqTgmW1h5Qu-jctRoMRYuvHf4xX9pw5TT2~LSzqKDtR0mjg38R~TXmLshaA9MgAoE7RyK873WOssIDG5MfRQ236zHRF9tZrkvuRgIxDAGL6O9x7hCoKsAg__"
+          alt="Moyo logo"
+        />
+      </div>
+      <nav>
+        <ul class="navItems">
+          <li>
+            <a
+              href="#"
+              class="navLink is-active"
+            >
+              APPLE
+            </a>
+          </li>
+          <li>
+            <a
+              href="#"
+              class="navLink"
+            >
+              SAMSUNG
+            </a>
+          </li>
+          <li>
+            <a
+              href="#"
+              class="navLink"
+            >
+              SMARTPHONES
+            </a>
+          </li>
+          <li>
+            <a
+              href="#"
+              class="navLink"
+              data-qa="hover"
+            >
+              LAPTOPS & COMPUTERS
+            </a>
+          </li>
+          <li>
+            <a
+              href="#"
+              class="navLink"
+            >
+              GADGETS
+            </a>
+          </li>
+          <li>
+            <a
+              href="#"
+              class="navLink"
+            >
+              TABLETS
+            </a>
+          </li>
+          <li>
+            <a
+              href="#"
+              class="navLink"
+            >
+              PHOTO
+            </a>
+          </li>
+          <li>
+            <a
+              href="#"
+              class="navLink"
+            >
+              VIDEO
+            </a>
+          </li>
```

Comment:
HTML Formatting: Each level of nesting, including text, contained inside the element, requires 2-space offset. Also, it's better to add empty lines between multiline sibling blocks of HTML to add some 'air' and simplify reading. But don't add them between parent and child elements.